python_scripts/linear_regression_non_linear_link.py

```diff
@@ -177,13 +177,29 @@
 from sklearn.preprocessing import PolynomialFeatures
 
 polynomial_regression = make_pipeline(
-    PolynomialFeatures(degree=3),
+    PolynomialFeatures(degree=3, include_bias=False),
     LinearRegression(),
 )
 polynomial_regression.fit(data, target)
 target_predicted = polynomial_regression.predict(data)
 mse = mean_squared_error(target, target_predicted)
 
+# %% [markdown]
+#
+# In the previous cell we had to set `include_bias=False` as otherwise we would
+# create a column perfectly correlated to the `intercept_` introduced by the
+# `LinearRegression`. We can verify that this procedure is equivalent to creating
+# the features by hand up to numerical error.
+
+# %%
+polynomial_regression[0].fit_transform(data) - data_expanded
```

Comment:
Printing the all the element-wise differences for the features of the dataset generates a very large numpy array. We can summarize this with the l_inf norm:

```suggestion
np.abs(polynomial_regression[0].fit_transform(data) - data_expanded).max()
```

Reply:
Perfect, thanks!